docs/en/05_Contributing/07_Code_of_conduct.md

```diff
@@ -0,0 +1,56 @@
+# SilverStripe community code of conduct (DRAFT)
+
+These guidelines aim to be an aspirational ideal for how we should behave when interacting in the SilverStripe developer community and to aid in building great open source software.
+
+The code of conduct applies to all digital discussion channels linked of to from silverstripe.org and where community members engage in any face to face discussion.
```

Comment:
Could this be simplified to something like "This code of conduct should be applied to all discussions and interactions involving SilverStripe and its related resources" ?